### DISCUSSION TITLE
Chlorsync support

### DISCUSSION THREAD

**Discussion by dfiore1230** [ballle98/AqualinkD#67] [General]
Has anyone looked at using this for autopilot chlorsync devices ?

**Comment by ballle98** [comment]
It does not look like this device is controlled by Aquilink control panel.  It does not look like it supports a RS485 serial connection.  It seems to have it's own local controls and wi-fi API

https://github.com/sfeakes/AqualinkD/wiki#notes-on-panel-version-and-protocols-

**Comment by dfiore1230** [comment]
hmm, according to the manual on p2, it shows it is RS485 based.
https://www.aquacal.com/wp-content/uploads/LTP0160-PoolSync-Manual-6-22-20.pdf
